docker/README.md

```diff
@@ -0,0 +1,41 @@
+# Docker
+
+This folder contains all the docker images which you could use to make your life easier.
+
+### *testrunner*
+This image basically just executes the tests of black-screen.
+
+It doesn't contain the source code, so you need to mount the git repo at **/black-screen**
+
+##### Usage
+
+**Change into the testrunner directory**
+
+- build the container with ```docker build -t black-screen_testrunner . ```
+- first startup: ```docker run --name bs-testrunner -v `pwd`/black-screen:/black-screen black-screen_testrunner ``` (Replace \`pwd\`/black-screen with your absolute path to the repo)
+- restart of tests: ```docker restart bs-testrunner; docker attach bs-testrunner```
```

Comment:
I think this section is unnecessary. As long as people can run the shell scripts, they shouldn't care about the implementation.